### ISSUE TITLE
Wii/GC banner is black for all games

### ISSUE THREAD

**Issue by eku** [Fledge68/WiiFlow_Lite#287]
**Describe the bug**
The banners of all GC and Wii games are animated, but you only see the first frame and then the video is black. Sound plays correctly.

**To Reproduce**
1. Just select on of the GC or Wii games in the cover flow
2. See first frame and than black video

**What version of WiiFlow Lite**
v5.4.9 works
v5.5.0 beta 11 does not work

**On Wii or Wii U vWii**
Wii

**Screenshots**
See attached video: [wiiflow_banner.zip](https://github.com/Fledge68/WiiFlow_Lite/files/7564973/wiiflow_banner.zip)

**Additional context**
Well, I'm not sure which beta of 5.5.0 broke this, cause i mostly play plugin games.
Works correct with USB Loader GX.

**Comment by Fledge68** [comment]
very strange cause they work fine for me.

keep in mind the banners are cached. is this occurring on no cached banners (first load of the banner)? mine are cached on SD.

**Comment by eku** [comment]
What do you suggest I should do? Deleting a folder? Which one?

**Comment by Fledge68** [comment]
hmmm so this looks like your GAME/TOOGLE_FULL_BTN is no longer a transparent image. when you view full screen banner's there a transparent image placed over top of it for the user to click on to shrink the banner to small view. check your rhapsodii shima ini for GAME/TOOGLE_FULL_BTN and see what the image is and then check to make sure that image is ok. but strange you say 5.4.9 is fine.

the only other thing i can  think of is for some reason 5.5.0 beta 11 didn't compile correctly on github. i may have to upload my copy for you to try. or you got a corrupt download.

**Comment by eku** [comment]
Thanks for pointing out the overlay button. I haven't changed anything in the theme, but I have optimised the PNGs with `mogrify -strip` (https://imagemagick.org/script/mogrify.php) . They still look like that on the PC afterwards, but wiiflow has problems with it. Interestingly, I do not observe this with the box, snapshot and cart covers and the background used by the theme.

In general, all files from `apps/wiiflow/imgs` are affected. Which makes me think that probably the files are read and interpreted differently from the others. Is the information helpful to you?

**Comment by Fledge68** [comment]
rhapsodii shima theme images were already optimized by @wiidev or someone else. you shouldn't have had to use mogrify. i would put the normal rhapsodii shima theme images back. it should solve your problem.

**Comment by eku** [comment]
I did optimize the WFL images and that causes the trouble.

**Comment by gingerbeardman** [comment]
Don't optimise the images, unless you are aware of the type of PNGs supported by WFL and can match that with your optimisation results.

Because the PNG library that WFL uses doesn't support all possible types of PNG. Notably monochrome and greyshade PNG, which is what transparent images are often optimised as.

TL;DR: use the bundled images.

**Comment by Fledge68** [comment]
yes exactly as gingerbeardman said. wfl probably doesn't support your png optimisation. just stick with the ones included with wfl.
